### PR TITLE
[PT-BR] Javascript Playgrounds.js, New Compiler Defaults.ts, Automatic Type Acquisition.js

### PR DESCRIPTION
**Pull request by RCarvalhoDan**

### CODE REVIEW

**Comment by RCarvalhoDan** [comment]
@khaosdoctor @alvarocamillont @danilofuchs @orta

File: packages/playground-examples/copy/pt/Playground/Config/JavaScript Playgrounds.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * Soma 2 números
+ * @param {number} O primeiro número
+ * @param {number} O segundo número
+ * @returns {number}
+ */
+function addTwoNumbers(a, b) {
+  return a + b;
+}
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Playground/Config/JavaScript Playgrounds.js]:
```suggestion
function somaDoisNumeros(a, b) {
  return a + b;
}
```

File: packages/playground-examples/copy/pt/Playground/Config/New Compiler Defaults.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
+//// { order: 1 }
+
+// Sim, nós sabemos, os padrões para o tsconfig do TypeScript não
+// tem strict ativado. Entretanto, em toda chance que temos o
+// time recomenda que os usuários tentem migrar para ter o modo
+// strict ativado nas suas configurações.
+
+// Porém, para o playground felizmente podemos estabelecer os
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Playground/Config/New Compiler Defaults.ts]:
```suggestion
// Porém, para o playground, felizmente podemos estabelecer os
```

File: packages/playground-examples/copy/pt/Playground/Language/Automatic Type Acquisition.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+// obtém definições de tipo do @types no npm nos bastidores para
+// conceder uma umelhor experiência de usuário para os usuários
+// de JavaScript.
+
+// O playground agora tem uma versão similar (porém um pouco
+// mais limitada ) do processo de aquisição de tipo intergrado
+// ao TypeScript.
+
+// Você pode utilizá-lo criando importações no seu código.
+// Funciona tanto através do @types de DefinitelyTyped ou por
```
Comment (khaosdoctor) [inline packages/playground-examples/copy/pt/Playground/Language/Automatic Type Acquisition.ts]:
```suggestion
// Funciona tanto através do @types do DefinitelyTyped ou por
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
Small changes

**Comment by RCarvalhoDan** [comment]
@khaosdoctor I added your suggestions.

**Review by khaosdoctor — APPROVED** [review]

**Comment by khaosdoctor** [comment]
lgtm

**Pull request merged**